addons/dark-www/addon.json

```
@@ -2297,6 +2297,25 @@
       "name": "\"Experimental\" Dark",
       "id": "experimentalDark",
       "description": "A dark theme by Maximouse based on a userstyle. Not experimental",
+      "values": {
+        "page": "#202020",
+        "navbar": "#855cd6",
+        "box": "#282828",
+        "gray": "#333333",
+        "blue": "#252c37",
```

```suggestion
        "blue": "#292d32",
```
I think the website and editor versions of Experimental Dark should stay as similar as possible. This color was changed in the editor version to be less saturated.